zerver/openapi/zulip.yaml

```diff
@@ -6648,6 +6651,23 @@ paths:
             type: boolean
             default: false
           example: true
+        - name: message_ids
+          in: query
+          description: |
+            A list of message IDs to fetch. Server will return messages matching
+            the narrow and the message IDs provided.
+
+            It is mutually exclusive with `anchor`, `num_before`
+            and `num_after`, i.e., if `message_ids` is provided, then `anchor`,
```

Comment:
Bump from https://github.com/zulip/zulip/pull/31787#discussion_r1781463688 re `include_anchor`.

Reply:
Sorry, I missed it somehow. Added to `include_anchor` to the mutually excluded list too.